src/gpuarray/ext_cuda.h

```diff
@@ -2,6 +2,7 @@
 #define LIBGPU_EXT_CUDA
 
 #include <cuda.h>
+#include <cuda_fp16.h>
```

Comment:
Well, if we have fp16-related code anywhere,  cuda_fp16_h should be thought of as extension of cuda.h